utils/cryptutils/cryptutils.go

```diff
@@ -432,6 +429,18 @@ func SavePrivateKeyToFile(fileName string, key crypto.PrivateKey) error {
 	return nil
 }
 
+// RSAPrivateKeyToPEM converts RSA private key to PEM format.
+func RSAPrivateKeyToPEM(out io.Writer, key *rsa.PrivateKey) error {
```

Comment:
It should be general-purpose function: key parameter should be crypto.PrivateKey type and this function should handle ecdsa key as well.